README.md

```diff
@@ -1,20 +1,39 @@
-# FormicRadar Revive
-# Download from the releases tab
+# FormicRadar Revive 📡
 
+![Project Banner](readme_banner.png?raw=true)
 
+![Static Badge](https://img.shields.io/badge/license-MIT-red?link=https%3A%2F%2Fopensource.org%2Flicense%2Fmit) ![Static Badge](https://img.shields.io/badge/mom_made-pizza_rolls-blue?link=https%3A%2F%2Fopensource.org%2Flicense%2Fmit)
+![GitHub forks](https://img.shields.io/github/forks/awdev1/FormicAcid-Radar-Revive)
+![GitHub Repo stars](https://img.shields.io/github/stars/awdev1/FormicAcid-Radar-Revive)
 
-## Usage
 
-Once installed and configured, you can use this to track virtual aircraft, manage air traffic, and enhance your flight simulation experience.
 
-## Contributing
 
-Contributions to FormicAcid Radar Revive are welcome! If you have any ideas, bug fixes, or feature enhancements, feel free to submit a pull request.
+> 📡 **The best radar tool** for managing PTFS Air Traffic.
 
-## Support
 
-For support or assistance, please open an issue on GitHub or contact the repository owner.
+> ✈️ This tool allows you to track & manage air traffic in PTFS and enhance your PTFS controlling experience.
 
-## License
+> 🌍 **Best used for ATC24** or Anything PTFS ATC!
 
-This project is licensed under the [MIT License](LICENSE).
+
+## Installation 🔧
+
+1. Get the latest [release](<https://github.com/awdev1/FormicAcid-Radar-Revive/releases>)
+2. Unzip the folder
```

Comment:
Un zip folder to a suitable folder ig

Reply:
Most likely not needed as the way windows unzips files

Reply:
Macs:

Reply:
Some software just extract files directly+ just some added protection to people who don't know how zips work